docs/sources/flow/_index.md

```diff
@@ -59,15 +59,15 @@ prometheus.remote_write "default" {
 
 ## Next steps
 
-* Read about how to [install and run][] Grafana Agent Flow.
 * Learn about the [core concepts][] of Grafana Agent Flow.
+* Follow our Grafana Agent Flow [Getting Started][] guides.
```

Comment:
nit - should they all starts with capital letters for links or none of them?

Reply:
I'll rework this. If the link is referencing the page title, it should match the page title casing.